src/gfx/process.cpp

```diff
@@ -680,7 +680,11 @@ class TileData {
 		for (uint32_t x = 0; x < 8; ++x) {
 			row <<= 1;
 			uint8_t index = palette.indexOf(tile.pixel(x, y).cgbColor());
-			assert(index < palette.size()); // The color should be in the palette
+			if (!(index < palette.size())) {
+				fprintf(stderr, "index %d >= palette.size() %d!\n",
+					index, palette.size());
+				exit(1);
+			}
```

Comment:
Assertions are only meant as internal consistency checks; why promote this?

Reply:
Oops, this is left over from trying to reproduce that rgbgfx bug report.